package.json

```
@@ -26,7 +26,8 @@
 		"version": "npm run generate:all && git add .",
 		"release": "np --no-cleanup --no-yarn --no-tests",
 		"update:check": "npx npm-check-updates -x np --dep dev,prod",
-		"update:commit": "npx npm-check-updates -u -x np,find-up --dep dev,prod && npm update && npm install"
+		"update:commit": "npx npm-check-updates -u -x np,find-up --dep dev,prod && npm update && npm install",
+		"install:config":  "get-git-folder https://github.com/kaiyuanshe/service-configuration main Polyfiller  ||  true"
```

```suggestion
		"install":  "get-git-folder https://github.com/kaiyuanshe/service-configuration main Polyfiller  ||  true"
```
用 NPM 预定义的脚本名会在相应的生命周期自动执行。